changelogs/fragments/5280-lxc_container-py3.yaml

```diff
@@ -0,0 +1,5 @@
+bugfixes:
+  - The ``lxc_container``` module has been updated to support Python 3 (https://github.com/ansible-collections/community.general/pull/5304)
+
+breaking_changes:
+  - The ``lxc_container`` module will no longer make any effort to support Python 2 (https://github.com/ansible-collections/community.general/pull/5304)
```

Comment:
Please note that breaking changes cannot be merged. This should probably be a `deprecated_feature`, since the module should still work fine with Python 2.

Reply:
done.